plugins/arRestApiPlugin/lib/exception/QubitApi404Exception.class.php

```diff
@@ -17,6 +17,6 @@
  * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
  */
 
-class QubitAPI404Exception extends Exception
+class QubitApi404Exception extends Exception
```

Comment:
Should we rename to `QubitApiNotFoundException` to be consistent with the other classes where the status code is never included?